src/common/SelectionList.js

```diff
@@ -0,0 +1,176 @@
+/* @flow strict-local */
+import React, { PureComponent } from 'react';
+import type { ComponentType } from 'react';
+import { FlatList, StyleSheet, View } from 'react-native';
+import type { ViewStyleProp } from 'react-native/Libraries/StyleSheet/StyleSheet';
+import type { NavigationScreenProp } from 'react-navigation';
+
+import type { Dispatch } from '../types';
+import type { ThemeColors } from '../styles';
+import { connect } from '../react-redux';
+import Label from './Label';
+import { IconDone, IconRight } from './Icons';
+import Touchable from './Touchable';
+import { BRAND_COLOR, ThemeContext } from '../styles';
+import { navigateToSelectionListOptionsScreen, navigateBack } from '../nav/navActions';
+import OptionDivider from './OptionDivider';
+import Screen from './Screen';
+
+export type OptionsType = $ReadOnly<{| [key: string]: { label: string } |}>;
```

Comment:
Let's make this like
```js
export type SelectionOptions<K> = Map<K, {| +label: string |}>;
```

That is:
* use a real `Map`, as discussed in previous comment
* use an exact object type
* name doesn't end in "Type" just because it's a type -- same reason we don't say `class OptionsScreenClass`, etc. Rather a name ending in "Type" means a type *of things which are themselves types*.
* "Options" is quite generic; add the word "Selection" from the name of this module.